interactive_engine/executor/common/dyn_type/src/object.rs

```diff
@@ -37,6 +38,10 @@ pub enum RawType {
     ULLong,
     Float,
     String,
+    DateFormat,
```

Comment:
`Date` instead of `DateFormat`, and applies to the follows.

Reply:
Done.

Reply:
Done.